src/openrct2/world/Map.cpp

```diff
@@ -1786,16 +1786,27 @@ int32_t map_get_highest_z(const CoordsXY& loc)
 
     auto z = surfaceElement->GetBaseZ();
 
-    // Raise z so that is above highest point of land and water on tile
+    // Raise z so that is above highest point of land
```

Comment:
I would suggest renaming this function to add land into the name. Make it more clear that this is now dedicated to land and the other function is dedicated to water.

Reply:
The other benefit of renaming is that it will show every user of the function and make it easier to confirm that all users are correctly modified.